apps/web/components/booking/BookingListItem.tsx

```diff
@@ -186,7 +189,9 @@ function BookingListItem(booking: BookingItemProps) {
     );
   };
 
-  const startTime = dayjs(booking.startTime).format(isUpcoming ? "ddd, D MMM" : "D MMMM YYYY");
+  const startTime = dayjs(booking.startTime)
+    .locale(language)
+    .format(isUpcoming ? "ddd, D MMM" : "D MMMM YYYY");
```

Comment:
have we ever tried use the localizedFormat dayjs plugin to display this?

![imagen](https://user-images.githubusercontent.com/525369/224789824-bb7068d8-1c0f-4d42-a64d-53992524468d.png)